src/note-components/notepage/note.css

```diff
@@ -1,109 +1,162 @@
 @import url('https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,300;0,400;0,500;0,900;1,300&display=swap');
-body{
+
+body {
     font-family: 'Montserrat', sans-serif;
+    margin-bottom: 3.125rem;
 }
-.main-container{
+
+.main-container {
     display: flex;
 }
-.left-nav{
+
+/* Sidebar CSS */
+.left-nav {
+    text-align: center;
     width: 20%;
     align-self: auto;
     box-shadow: 2px 2px 6px 0px rgba(0, 0, 0, 0.1);
 }
-.right-cont{
-    width: 80%;
-    margin-top: 30px;
-    display: flex;
-    justify-content: center;
-    align-items: center;
-    flex-direction: column;
-    gap: 20px;
-}
-.left-list{
+
+.left-list {
     list-style: none;
     font-size: larger;
     padding-left: 5px;
 }
-.left-item{
-    margin: 15px;
+
+.left-item {
+    margin: 0.9375rem;
     padding: 10px;
     border-radius: 5px;
     color: var(--primary-color);
 }
-.left-item:hover{
+
+.left-item:hover {
     background-color: #b6bfc9;
```

Comment:
Use CSS variable for color

Reply:
This color was used just once, that is why I didn't make a variable out of it.